docs/content/integration/openid-connect/odoo/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Odoo"
+description: "Integrating Odoo with the Authelia OpenID Connect 1.0 Provider."
+summary: ""
+date: 2024-03-24T09:09:0+01:00
+draft: false
+images: []
+weight: 620
+toc: true
+community: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Tested Versions
+
+* [Authelia]
+  * [v4.38.6](https://github.com/authelia/authelia/releases/tag/v4.38.6)
+* [Odoo]
+  * [17.0](https://github.com/odoo/odoo/tree/17.0)
+
+## Before You Begin
+
+{{% oidc-common %}}
+
+### Assumptions
+
+This example makes the following assumptions:
+
+* __Application Root URL:__ `https://odoo.example.com/`
+* __Authelia Root URL:__ `https://auth.example.com/`
+* __Client ID:__ `client_id_odoo`
+
+## Configuration
+
+### Authelia
+
+The following YAML configuration is an example __Authelia__
+[client configuration](../../../configuration/identity-providers/openid-connect/clients.md) for use with [Odoo]
+which will operate with the above example:
+
+```yaml
+identity_providers:
+  oidc:
+    clients:
+      - client_id: 'client_id_odoo'
+        client_name: 'odoo.example.com'
+        public: 'true'
+        redirect_uris:
+          - 'https://odoo.example.com/auth_oauth/signin'
+        scopes:
+          - 'openid'
+          - 'email'
+          - 'profile'
+        response_types:
+          - 'token'
```

Comment:
What happens here if you use code? Seems based on their docs like they support the auth code flow and it's much more secure than the implicit flows.

Reply:
I'm completely new in openid and other sso stuffs....
For sure, if I only set the response_types to code, it does not work.
It works if I set code and token or token only.

I can't say more :)

On their doc about Azure integration, they talk about these implicit flows (cf https://www.odoo.com/documentation/17.0/applications/general/users/azure.html)

Reply:
Yeah it's okay, your contribution is valuable because I can't every app, but I know a lot about the OAuth 2.0 and OpenID Connect 1.0 specs which is where I can be useful.